interfaces/Portalicious/src/app/components/registration-page-layout/registration-page-layout.component.ts

```diff
@@ -64,6 +85,47 @@ export class RegistrationPageLayoutComponent {
     ),
   );
 
+  readonly actionMenuItems = computed<MenuItem[]>(() => [
+    {
+      items: [
+        {
+          label: $localize`Add note`,
+          icon: 'pi pi-pen-to-square',
+          command: () => {
+            this.addNoteFormVisible.set(true);
+          },
+          visible: this.canUpdatePersonalData(),
+        },
+        this.registrationMenuService.createContextItemForMessage({
+          projectId: this.projectId(),
+          command: () => {
+            this.sendMessage();
+          },
+        }),
+      ],
+    },
```

Comment:
```suggestion
        {
          label: $localize`Add note`,
          icon: 'pi pi-pen-to-square',
          command: () => {
            this.addNoteFormVisible.set(true);
          },
          visible: this.canUpdatePersonalData(),
        },
        this.registrationMenuService.createContextItemForMessage({
          projectId: this.projectId(),
          command: () => {
            this.sendMessage();
          },
        }),
```

I see extra whitespace in the menu, and I think it's because of this extra level of nesting that shouldn't be necessary.

Reply:
If you remove that nesting it sort of converts Add Note and Send Message to the subheader style
![image](https://github.com/user-attachments/assets/57241e7f-9a31-4613-9ff7-06e82a8ef06d)
I am not too annyed about this extra white space, but maybe it can be removed with some custom css?

Reply:
The extra white space is really ugly 😄  And I wouldn't introduce extra CSS unless necessary - that feels like a hacky workaround.

I see that the issue you're encountering is a somewhat known bug https://github.com/primefaces/primeng/issues/17131

Maybe we can indeed used [TieredMenu](https://primeng.org/tieredmenu#popup) instead? And put the status update group under a tier. If that doesn't work, then I would rather
- remove the "status update" category
- just add a separator between the add note & message actions, and the other actions (like we have in the registrations table)
- add a comment about what we would've like to do, but can't, and link to the bug above

Reply:
We chose to add a label to the first 2 nested items called 'General'